wordpress-mobile/WordPress-iOS
# Nuking AFNetworking: Notifications Media Downloader

### Details:
This PR replaces AFNetworking with plain URLSession usage.

### To test:
1. You'll need two WordPress.com accounts: **A** and **B**
2. Log into **A** and post a comment into any of **B**'s posts
3. Log into **B** (on the  web) and edit the comment (via wp-admin). You can paste this snippet:
```
Test <img src="https://lanteanar.files.wordpress.com/2017/12/image.jpg">
```
4. Launch WPiOS in **B**'s account
5. Open Notifications List: find the new Notification

Verify the image shows up correctly!

cc @SergioEstevao 
Thanks in advance!!
